libcodechecker/libhandlers/analyze.py

```diff
@@ -177,6 +184,58 @@ def add_arguments_to_parser(parser):
                                help="File containing argument which will be "
                                     "forwarded verbatim for Clang-Tidy.")
 
+    if is_ctu_capable():
+        ctu_opts = parser.add_argument_group('cross translation unit analysis')
+        ctu_mutex_group = ctu_opts.add_mutually_exclusive_group()
+        ctu_mutex_group.add_argument('--ctu',
+                                     action='store_const',
+                                     const=[True, True],
+                                     dest='ctu_phases',
+                                     default=argparse.SUPPRESS,
+                                     help="Perform cross translation unit "
+                                          "(CTU) analysis (both collect and "
+                                          "analyze phases) using default "
+                                          "<ctu-dir> for temporary output. "
+                                          "At the end of the analysis, the "
+                                          "temporary directory is removed.")
+        ctu_opts.add_argument('--ctu-dir',
+                              metavar='<ctu-dir>',
+                              default=os.path.join(
+                                  util.get_default_workspace(),
+                                  'ctu-dir'),
+                              help="Optional parameter defines the temporary "
+                                   "directory used between CTU phases.")
+        ctu_mutex_group.add_argument('--ctu-collect-only',
+                                     action='store_const',
+                                     const=[True, False],
+                                     dest='ctu_phases',
+                                     default=argparse.SUPPRESS,
+                                     help="Perform only the collect phase of "
+                                          "CTU. Keep <ctu-dir> for further "
+                                          "use.")
+        ctu_mutex_group.add_argument('--ctu-analyze-only',
```

Comment:
Instead of negative option strings (that turn out feature X and leave only Y on, **while** the default is both turned off...), we should use positive ones, such as `--ctu-run-analyze`.